shell/packages/sandstorm-db/db.js

```diff
@@ -639,6 +639,30 @@ SetupSession = new Mongo.Collection("setupSession");
 //   creationDate: Date object indicating when this session was created.
 //   hashedSessionId: the sha256 of the secret session id that was returned to the client
 
+const DesktopNotifications = new Mongo.Collection("desktopNotifications");
+// Responsible for very short-lived queueing of desktop notification information.
+// Entries are removed when they are ~30 seconds old.  This collection is a bit
+// odd in that it is intended primarily for edge-triggered communications, but
+// Meteor's collections aren't really designed to support that organization.
+// Fields for each :
+//
+//   _id: String.  Used as the tag to coordinate notification merging between browser tabs.
+//   creationDate: Date object. indicating when this notification was posted.
+//   accountId: String. Account id to which this notification was published.
```

Comment:
> With the exception of ApiTokens

`ApiTokens` does not use `accountId` to identify to whom the token belongs -- and in fact, ApiTokens never belong to accounts. So my original statement is correct. :)

> It would probably be wise to update the other comments where userId is used to say "the accountId of the user" rather than "the user"

I agree.